lib/stoplight/infrastructure/data_store/redis.rb

```diff
@@ -316,6 +316,20 @@ def transition_to_color(config, color)
           became_red == 1
         end
 
+        # Removes all traces of a light from Redis metadata (metrics will expire by TTL).
+        #
+        # @param config [Stoplight::Domain::Config] The light configuration.
+        # @return [Integer] number of keys removed
+        def delete_light(config)
+          @redis.then do |client|
+            client.evalsha(
+              delete_light_sha,
```

Comment:
I think we could just use `client.del(metadata_key(config))` in this case. We use scripts when atomicity is needed and/or an operation requires multiple steps; in this case, this is unnecessary.